recipes/bioconda-utils/meta.yaml

```diff
@@ -20,10 +20,10 @@ build:
 
 requirements:
   host:
-    - python >=3.7
+    - python
```

Comment:
Removed version, so I believe it's now properly using the pinned python version (3.10).